azure-app-service/src/main.tf

```diff
@@ -1,7 +1,3 @@
-locals {
-  docker_registry_url = strcontains(var.image.name, "azurecr.io") ? "https://mcr.microsoft.com" : "https://index.docker.io"
```

Comment:
If there are only 2 options, can we not just infer this? Why does this logic need to change - what changed on Azure's end?

Reply:
I was mistaken about `azurecr.io` = `mcr.microsoft.com`, they are actually different registries. And app service can use an image from any public registry, i.e. `massdriverwestus.azurecr.io`, `mcr.microsoft.com`, `docker.io`, even ECR and GCR as long as they are public